src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -225,7 +226,8 @@ void FixupParametersFromDocs (XElement api)
 		AndroidDocScraper CreateDocScraper (string dir)
 		{
 			switch (DocletType) {
-			default: return new DroidDocScraper (dir);
+			default: return new DroidDoc2Scraper (dir);
+			case JavaDocletType.DroidDoc: return new DroidDocScraper (dir);
```

Comment:
Shouldn't there also be a `case JavaDocletType.DroidDoc2:` entry, so that this can be explicitly selected?